src/index.js

```diff
@@ -136,7 +136,7 @@ function dest(destination, opts) {
       );
     }
 
-    stream.pipe(symdest(destination)).pipe(result);
+    stream.pipe(vfs.dest(destination)).pipe(result);
```

Comment:
Considering that the stream is es.MapStream, I'm not sure whether it can be directly piped to vfs like this, or whether there is a way to convert the stream objects to Vinyl files, first.

Reply:
Being es.MapStream is OK. In gulp all streams just contain Vinyl instances.

Gulp symdest was created at a time that vinyl-fs didn't support symlinks. I believe it started supporting it after a while. Please test this change end-to-end with VS Code's build by running `yarn gulp vscode-darwin-x64` (or `-arm64`) and ensuring the build it produces can be executed.

Reply:
Looks like CI is failing now for 14.x. Should I restrict CI to run only for 16.x and up?

Reply:
code-oss on Windows x64 produced by `yarn gulp vscode-win32-x64` followed by `yarn gulp vscode-win32-x64-archive` is sane.

Reply:
Windows isn't relevant here: symlinks don't exist in this shape in Windows. And even though symlinks do exist in Linux, we don't depend on them to produce a valid VS Code Linux build.

But we **do** depend on symlinks to produce a valid macOS build. Please make sure a valid macOS build can be built.

Reply:
Feel free to remove Node 14.x from the CI.

Reply:
I removed Node 14.x from the CI and replaced Node 17.x with Node 18.x, so CI looks better.

Running the macOS gulp command worked fine on my machine, though I wasn't sure where the build ended up after, so I ran the pipeline on a test branch, and it looks like sanity testing fails, so this PR isn't ready to merge.
https://dev.azure.com/monacotools/Monaco/_build/results?buildId=213013&view=results

Reply:
Running `yarn gulp vscode-darwin-x64` locally will produce a build in a folder called `VSCode-darwin-x64`, sibling to the `vscode` folder.

Reply:
Got it. The PR seems to cause a regression because it generates a file called `LICENSES.Chromium.html` outside of/next to `Code - OSS.app`, whereas it previously did not.

Reply:
Let me know how e2e testing with building VS Code?

Reply:
Closing for now until I find a better fix. I believe this PR causes a regression.